src/Bindings/DeprecatedBindings.cpp

```diff
@@ -292,6 +292,58 @@ static int tolua_AllToLua_StringToMobType00(lua_State* tolua_S)
 
 
 
+static int tolua_get_cItem_m_Lore(lua_State * tolua_S)
+{
+	// Maintain legacy m_Lore variable as Lore table split by ` (grave-accent)
+	cLuaState L(tolua_S);
+	if (!L.CheckParamSelf("cItem"))
+	{
+		return 0;
+	}
+
+	cItem * Self = nullptr;
+	L.GetStackValue(1, Self);
+
+	AString LoreString = StringJoin(Self->m_LoreTable, "`");
+
+	L.Push(LoreString);
+
+	LOGWARNING("Warning in read from variable 'cItem.m_Lore': m_Lore is deprecated. Please use m_LoreTable");
```

Comment:
Too wordy. `cItem.m_Lore is deprecated, use cItem.m_LoreTable instead` is enough.